workspace_tools/targets.py

```diff
@@ -1269,18 +1269,21 @@ def __init__(self):
 class RBLAB_BLENANO(MCU_NRF51_16K):
     def __init__(self):
         MCU_NRF51_16K.__init__(self)
+        self.supported_toolchains = ["GCC_ARM"]
```

Comment:
should I add ARM too, since it seems to be part of `workspace_tools/build_release.py`, but I can't actually test that.